lua/chainsaw/core/log-commands.lua

```diff
@@ -104,5 +104,38 @@ function M.removeLogs()
 	vim.b.timelogStart = nil
 end
 
+function M.removeLogsVisual()
+	local marker = require("chainsaw.config.config").config.marker
+	local numOfLinesBefore = vim.api.nvim_buf_line_count(0)
+
+	-- Get the start and end of current visual selection
+	local selectedStart = vim.fn.getpos("v")
+	local selectedEnd = vim.fn.getpos(".")
```

Comment:
Just a bit, but directly storing `[2]` here would make the code more readable; `[1]` is never accessed after all